create_cluster.py

```diff
@@ -426,6 +431,7 @@ def launch_instance(region: str, ip: dict, ami: object, subnet_id: str,
             SubnetId=subnet_id,
             PrivateIpAddress=ip['PrivateIp'],
             BlockDeviceMappings=block_devices,
+            IamInstanceProfile=instance_profile_name,
```

Comment:
This is not going to work.  You need a dict with `Arn` or `Name`, see: http://boto3.readthedocs.io/en/latest/reference/services/ec2.html#EC2.Client.run_instances